src/components/MapView/MapView.tsx

```diff
@@ -210,7 +219,8 @@ const MapView = forwardRef<MapViewHandle, ComponentProps>(
                         defaultSettings={defaultSettings}
                         // Include centerCoordinate here as well to address the issue of incorrect coordinates
                         // displayed after the first render when the app's storage is cleared.
-                        centerCoordinate={centerCoordinate}
+                        centerCoordinate={interactive ? centerCoordinate : undefined}
```

Comment:
Can we just add this logic into `centerCoordinate` useMemo? Is there any reason to add ternary operator here instead?

Reply:
@hungvu193 I fixed it in [commit](https://github.com/Expensify/App/pull/43630/commits/d0bcc2be0828f26febb41e19ed5f48980a941973).